.github/workflows/ci.yml

```
@@ -143,7 +143,7 @@ jobs:
         run: docker logs ocean_aquarius_1 && docker logs ocean_provider_1 && docker logs ocean_provider2_1 && docker logs ocean_computetodata_1
         if: ${{ failure() }}
       - name: Upload coverage
-        uses: actions/upload-artifact@v2
+        uses: actions/upload-artifact@v4
         with:
           name: coverage
```

We're uploading three artifacts all with the same name and you're getting a name conflict error. 

I suggest changing this one:

```suggestion
          name: coverage-integration
```



And change the one for the unit test to `coverage-unit`